Apps/Examples/Examples/AppDelegate.swift

```diff
@@ -60,3 +63,9 @@ final class SceneDelegate: NSObject, UISceneDelegate {
         windows.append(window)
     }
 }
+
+private func enableTracing() {
+    // Enable tracing in iOS sdk and GLNative by default in Examples.
+    let settings = SettingsServiceFactory.getInstance(storageType: SettingsServiceStorageType.nonPersistent)
+    _ = settings.set(key: "com.mapbox.tracing", value: ["com.mapbox.maps": "platform"])
```

Comment:
Can we have that setting in arguments of Examples scheme? So it would be `enableTracingIfNeeded` function that checks the state of some custom `UserDefaults` key (like `EnableTracing`)